tests/integration/staking/keeper/genesis_test.go

```diff
@@ -230,11 +243,24 @@ func TestInitGenesisLargeValidatorSet(t *testing.T) {
 	assert.NilError(t, err)
 
 	abcivals := make([]abci.ValidatorUpdate, 100)
+	validatorUpdates := make([]module.ValidatorUpdate, len(abcivals))
 	for i, val := range validators[:100] {
 		abcivals[i] = val.ABCIValidatorUpdate(f.stakingKeeper.PowerReduction(f.sdkCtx))
```

Comment:
Do we have to go via the abci struct in PowerReduction?

Reply:
done! use `ModuleValidatorUpdate`